app/views/group_offers/show.html.slim

```diff
@@ -68,23 +68,20 @@ h1= @group_offer.title
             td= t_attr(:schedule_details)
             td= @group_offer.schedule_details
 
-- if !@group_offer.external? && @group_offer.responsible
-  - if @group_offer.volunteer_responsible_state?
-    h3= t("volunteer_responsible_state.#{@group_offer.volunteer_responsible_state}")
-  - else
-    h3= t_attr(:responsible)
-  .row
-    .col-xs-12
-      ul
-        li= link_to @group_offer.responsible, volunteer_path(@group_offer.responsible)
-
-- if @group_offer.volunteers.any?
+- if @group_offer.group_assignments.any?
   h3= t_attr(:volunteers)
   .row
```

Comment:
remove this row and try to ad the row class to the `li.row` at line 76.
Pretty sure that would work.

Reply:
`.row` is indeed not needed but `li.row` breaks the alignment between the dot and the list item.